src/Cli/dotnet/CommonOptions.cs

```diff
@@ -213,6 +215,24 @@ public static Option<bool> CurrentRuntimeOption(string description) =>
             HelpName = CliStrings.VersionSuffixArgumentName
         }.ForwardAsSingle(o => $"--property:VersionSuffix={o}");
 
+    public static Option<NuGetVersion?> VersionOption =
+        new Option<NuGetVersion?>("--version")
+        {
+            Description = CliCommandStrings.PackCmdVersionDescription,
+            HelpName = CliCommandStrings.PackCmdVersion,
+            CustomParser = r =>
+            {
+                if (r.Tokens.Count == 0)
+                    return null;
```

Comment:
What's the default "arity" for an option?  Because, if it's `ZeroOrOne`, that would mean that `dotnet pack my.nuspec --version` is valid and pack should continue, even though no value was provided to version.

Reply:
Actually, I think there should explicitly be a unit test for this. At least, I'd ask for one if it was in the NuGet.Client repo

Reply:
Yes - arity should probably be ExactlyOne here. With that, you have a framework-provided guarantee that you will only have one token in the provided input array.

Reply:
Default Arity is dependent on the type of the option - certain types with well defined defaults (mostly Boolean flags) get an Arity of ZeroOrOne and an implicit value of true when specified without a value.